jina/parsers/__init__.py

```diff
@@ -3,6 +3,8 @@
 from jina.parsers.orchestrate.runtimes.container import mixin_container_runtime_parser
 from jina.parsers.orchestrate.runtimes.head import mixin_head_parser
 
+GATEWAY_NAME = 'gateway'
```

Comment:
add this to some `helper` and use it everywhere in the code

Reply:
done